src/ai/backend/manager/models/gql.py

```diff
@@ -579,6 +579,10 @@ class Queries(graphene.ObjectType):
         is_operation=graphene.Boolean(
             deprecation_reason="Deprecated since 24.03.4. This field is ignored if `load_filters` is specified and is not null."
         ),
+        load_only_active=graphene.Boolean(
+            default_value=True,
+            description="Added in 25.3.1.",
+        ),
```

Comment:
It seems better for the API to provide a way to filter types by receiving an enumeration list rather than as a boolean.

Reply:
Resolved by replacing `load_only_active` with `filter_by_statuses`.